src/resources/i18n/translations/en.json

```
@@ -260,20 +260,20 @@
   "Custom API domain": "Custom API domain",
   "Custom Feeds": "Custom Feeds",
   "Custom RSS feeds to parse in parallel": "Custom RSS feeds to parse in parallel",
-  "Custom RSS feeds to parse in parallel helper text": "Parsing too many feeds in parallel may cause issues on some devices. 3 is recommended.",
+  "Custom RSS feeds to parse in parallel helper text": "Parsing more than three feeds at once may cause issues on some devices.",
   "Custom web domain": "Web domain",
-  "Custom web domain subtext": "Custom domains to use an alternate server for your app data.",
+  "Custom web domain subtext": "These allow choice of app data server.",
```

```suggestion
  "Custom web domain subtext": "Allows using different servers to fetch app data.",
```
Not sure if this is better. I think it will wrap.